### DISCUSSION TITLE
VALENCE

### DISCUSSION THREAD

**Discussion by jacekkozuch** [TinkerTools/tinker-gpu#224] [Q&A]
Dear all,

I wrote already in the Tinker8 discussion page, since it's a general Tinker question, but here once more: 

I wanted to ask what the current state of the VALENCE program is. I remember that I used it once in a < 8.0 version of Tinker, but I've seen in the documentation that "This program is still under development."

How does it work currently and what can I do with it at the current state?
I remember that I need a Gaussian output file which contains a frequency calculation. Any specifics that I should take care about (i.e. out or chk or fchk file, g09 or g16, etc)? I'm aiming at a Fe-complex, so I hope that the Fe atom won't be a problem.

I appreciate any advice and support!

Thank you in advance!
Best,
Jacek

**Comment by jayponder** [comment]
[Same comment also posted in Tinker discussion.]

Hi Jacek, We do not recommend using the VALENCE program. The code has not really been looked at in a very long time, and there are better tools now available for parameterization. If you are looking for AMOEBA parameters for a system, you should try the Poltype program, which is also available from the TinkerTools GitHub site. If you want parameters for a standard partial charge force field, then the ForceBalance program from Lee-Ping Wang's lab could be a reasonable alternative. That said, transition metal systems are still hard for most force fields, including AMOEBA. So you may need to come up with parameters involving the iron atom separately.

If all you really want to do is get bond and angle parameters that match QM vibrational frequencies, then you might be able to try VALENCE. Its use and behavior are probably no different from what you would have done years ago using the program in a much earlier version of Tinker.

**Reply by jacekkozuch** [reply]
Hi 

First, thanks for the response. Unfortunately Poltype is not working currently with metals and I'm currently looking into how deep inside the code I would have to modify things. 
However, I have my multipole, polarize and vdw parameters already and I only need to get bond, angle and torsions, or at least a first guess, so I can refine things manually going forward. As a test, I tried VALENCE with water:

`valence 1 water.xyz -k water.key water_freq.fchk`
or 
`valence 1 water.xyz -k water.key water_freq.out`

but I am getting a "VALENCE  --  The Number of Atoms is Not Consistent" error (see water.zip with the files I am using). 
Maybe I doing something obviously wrong or it is an issue with Gaussian16 outputs?

Thanks in advance!

[water.zip](https://github.com/TinkerTools/tinker9/files/11412528/water.zip)
